test/specs/utils/tablify.js

```diff
@@ -57,12 +57,16 @@ describe('tablify util', () => {
 	Given('a deeply nested object', given.aDeeplyNestedObject, () => {
 		When('the object is tablified', when.theObjectIsTablified, () => {
 			Then(
-				'the returned table should have a head with the object’s deeply nested keys',
-				then.theReturnedTableShouldHaveAHeadWithTheObjectDeeplyNestedKeys,
+				'it should throw a print error "Maximum nesting of the object is exceeded to be displayed as a table."',
```

Comment:
`it should throw print error` to match function name.

Reply:
it was actually like that and I changed it last minutes ='(